tests/metrics/verifyPostgresql_test.js

```diff
@@ -0,0 +1,56 @@
+const pmmManagerCmd = 'bash /srv/pmm-qa/pmm-tests/pmm-framework.sh --pmm2';
+
+Feature('Test functional related to the PostgreSQL');
+
+Before(async ({ I }) => {
+  await I.Authorize();
+});
+
+Scenario(
+  'PMM-T1032 - Verify default PG queries are shipped with PMM',
+  async ({ I, grafanaAPI }) => {
+    const metricNames = [
+      'pg_replication_lag',
+      'pg_postmaster_start_time_seconds',
+      'pg_stat_user_tables_analyze_count',
+      'pg_statio_user_tables_heap_blks_hit',
+      'pg_database_size_bytes',
+    ];
+    const serviceName = 'PG-default-queries';
+
+    await I.verifyCommand(`${pmmManagerCmd} --addclient=pdpgsql,1 --pdpgsql-version=13.4 --deploy-service-with-name ${serviceName}`);
+    metricNames.forEach((metric) => {
+      grafanaAPI.checkMetricExist(metric, { type: 'service_name', value: serviceName });
+    });
+    await I.verifyCommand(`${pmmManagerCmd} --cleanup-service ${serviceName}`);
+  },
+);
+
+Scenario(
+  'PMM-T1102 - Verify last scrape of metrics from PostgreSQL',
+  async ({ I, grafanaAPI, settingsAPI }) => {
+    const metricName = 'pg_up';
+    const serviceName = 'PG-service';
+
+    // fix of the scraping bug
+    await settingsAPI.changeSettings({ metrics_resolutions: { lr: '60s', mr: '15s', hr: '10s' } }, true);
```

Comment:
where do you reset this back?

Reply:
Settings are restored at line 54: `await settingsAPI.restoreSettingsDefaults();`